core/src/test/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGeneratorTest.scala

```diff
@@ -0,0 +1,57 @@
+package com.databricks.labs.remorph.generators.sql
+
+import com.databricks.labs.remorph.parsers.intermediate.{InnerJoin, JoinDataType, RightOuterJoin}
+import com.databricks.labs.remorph.parsers.{intermediate => ir}
+import org.scalatest.wordspec.AnyWordSpec
+
+class LogicalPlanGeneratorTest extends AnyWordSpec with GeneratorTestCommon[ir.LogicalPlan] with ir.IRHelpers {
+
+  override protected val generator = new LogicalPlanGenerator
+
+  "Project" should {
+    "transpile to SELECT" in {
+      ir.Project(namedTable("t1"), Seq(ir.Id("c1"))) generates "SELECT c1 FROM t1"
+      ir.Project(namedTable("t1"), Seq(ir.Star(None))).doesNotTranspile
+    }
+  }
```

Comment:
```suggestion
  ir.Project(namedTable("t1"), Seq(ir.Id("c1"))) shouldGenerate "SELECT c1 FROM t1"
  ir.Project(namedTable("t1"), Seq(ir.Star(None))) shouldFail
```

@vil1 i wonder if we can have this kind of test DSL?... this way we can run all examples in parallel and re-run only those that failed... though it might be too much